website/public/locales/jp/labelling.json

```diff
@@ -0,0 +1,79 @@
+{
+  "accurate": "Точное",
```

Comment:
for the japanese files, if we don't have a translation now, then please use the english one, or remove entirely.